assets/js/googlesitekit/data/create-settings-store.js

```diff
@@ -61,12 +62,13 @@ const ROLLBACK_SETTINGS = 'ROLLBACK_SETTINGS';
  * @since 1.6.0
  * @private
  *
- * @param {string} type                 The data to access. One of 'core' or 'modules'.
- * @param {string} identifier           The data identifier, eg. a module slug like 'search-console'.
- * @param {string} datapoint            The endpoint to request data from, e.g. 'settings'.
- * @param {Object} options              Optional. Options to consider for the store.
- * @param {number} options.storeName    Store name to use. Default is '{type}/{identifier}'.
- * @param {Array}  options.settingSlugs List of the slugs that are part of the settings object
+ * @param {string} type                         The data to access. One of 'core' or 'modules'.
+ * @param {string} identifier                   The data identifier, eg. a module slug like 'search-console'.
+ * @param {string} datapoint                    The endpoint to request data from, e.g. 'settings'.
+ * @param {Object} options                      Optional. Options to consider for the store.
+ * @param {Array}  [options.ownedSettingsSlugs] Optional. List of "owned settings" for this module, if they exist.
+ * @param {number} [options.storeName]          Store name to use. Default is '{type}/{identifier}'.
+ * @param {Array}  [options.settingSlugs]       List of the slugs that are part of the settings object
  *                                      handled by the respective API endpoint.
```

Comment:
This continuation of the description is now misaligned -
```suggestion
 *                                              handled by the respective API endpoint.
```

Reply:
Huh, weird my ESLint didn't catch that. Fixed! 👍🏻